lxd/networks.go

```diff
@@ -755,12 +755,14 @@ func networkLeasesGet(d *Daemon, r *http.Request) response.Response {
 					if err != nil {
 						continue
 					}
-				} else if inst.Type() == instancetype.VM {
-					d, err = inst.(*vmQemu).fillNetworkDevice(k, d)
+				}
+				/* tomp TODO why hasn't this been done at instance/device create time?
```

Comment:
@stgraber I wanted to ask you about this part, I didn't want to add `fillNetworkDevice` as an exported function in the `Instance` interface, and wondered why we need to call `fillNetworkDevice` here, as it shouldnt the MAC address be generated when the device or instance is created?

Reply:
`fillNetworkDevice` is still needed to populate `d["hwaddr"]` from the relevant volatile key.